tests/unit/s2n_client_psk_extension_test.c

```diff
@@ -72,6 +72,41 @@ static int mock_time(void *data, uint64_t *nanoseconds)
     return S2N_SUCCESS;
 }
 
+static int s2n_setup_ticket_key(struct s2n_config *config)
```

Comment:
We could use `S2N_RESULT` instead of `int` return type too

Reply:
If you call S2N_BLOB_FROM_HEX you have to use an int return value.